apps/ledger-live-mobile/src/components/CustomImage/ImageResizer.tsx

```diff
@@ -0,0 +1,58 @@
+import { useEffect } from "react";
+import { manipulateAsync, SaveFormat } from "expo-image-manipulator";
+import { ImageBase64Data, ImageDimensions, ImageFileUri } from "./types";
+import { ImageResizeError } from "./errors";
+
+export type ResizeResult = ImageBase64Data & ImageDimensions;
+
+export type Props = ImageFileUri & {
+  targetDimensions: ImageDimensions;
+  onError: (error: Error) => void;
+  onResult: (res: ResizeResult) => void;
+};
+
+/**
+ * Component to resize an image.
+ *
+ * It:
+ *  - takes as an input an image file URI & target dimensions as an input
+ *  - renders nothing
+ *  - outputs the resulting image as a base64 data URI.
+ *
+ * */
+const ImageResizer: React.FC<Props> = props => {
```

Comment:
As this component is not rendering anything, and it has a function (resizing the image), we should transform it in a custom hook ? And call this hook directly in `Step2Preview` screen ? WDYT ? 
And maybe in the future it could be in live-common if we do the same thing in LLD (well it will depend if [ImageManipulator](https://docs.expo.dev/versions/latest/sdk/imagemanipulator/) works on Electron - but that's not for now)